js/script.js

```diff
@@ -48,15 +48,16 @@
   };
```

Comment:
Don't forget your semi-colons when you're ending/ closing a line. In this example it is line 47 (return elTr; ).